### ISSUE TITLE
Text editors for collection description blocks should allow rich text

### ISSUE THREAD

**Issue by vantuyls** [OregonDigital/OD2#426]
### Descriptive summary

need the ability to use rich text functionality in description fields on Collection landing pages to allow for: 
- formatting
- insertion of images
- linking out

**Comment by raybrarian** [comment]
Hyrax currently has a WYSIWYG editor - needs more advanced text options.

**Comment by CGillen** [comment]
The question for you @kmthorn and @petersec  is if we still want rich text on description fields

**Comment by kmthorn** [comment]
Yes. This is needed -- specifically to create embedded hyperlinks.
